udata/routing.py

```diff
@@ -149,12 +149,15 @@ def to_url(self, obj):
                 slug=territory.slug)
 
         code = getattr(obj, 'code', None)
-        validity = getattr(obj, 'validity', None)
         slug = getattr(obj, 'slug', None)
-        if code and validity and slug:
+        validity = getattr(obj, 'validity', {})
+        if code and slug:
             return '{level_name}/{code}@{start_date}/{slug}'.format(
-                level_name=level_name, code=code, start_date=validity['start'],
-                slug=slug)
+                level_name=level_name,
+                code=code,
+                start_date=validity.get('start', 'latest'),
```

Comment:
I think it need some clarification in another PR. Right now, this PR ensure it has the only supported behavior.